networkml/helpers/pandas_csv_importer.py

```diff
@@ -95,15 +101,26 @@ def recast_df(df):
 
 def import_csv(in_file):
     # We need converters, so we can't use dtypes parameter, and that results in an un-suppressable warning.
-    csv_fields = set(pd.read_csv(
-        in_file, index_col=0, nrows=0).columns.tolist())
+    sample_df = pd.read_csv(in_file, index_col=0, nrows=100)
```

Comment:
Is there a specified reason for moving from `nrows=0` to `nrows=100`? According to [the docs](https://pandas.pydata.org/pandas-docs/dev/reference/api/pandas.read_csv.html) this is an optional parameter. If there isn't a compelling reason to have it at either value i think it makes mor sense to omit it.

Reply:
Apologies yes. nrows=100 allows us to take a sample of the dataframe to see what types are in it (previously we didn't need to do that, we just need the column header, which is why it was nrows=0).

Reply:
that makes sense.